packages/ionic-angular/package.json

```diff
@@ -26,10 +26,10 @@
   },
   "dependencies": {
     "@nxext/capacitor": "^15.7.0",
-    "@nrwl/devkit": "15.7.0",
-    "@phenomnomnominal/tsquery": "4.1.1"
+    "@phenomnomnominal/tsquery": "4.1.1",
+    "@nx/devkit": "16.0.0-beta.7"
   },
   "peerDependencies": {
-    "@nrwl/angular": "^15.7.0"
+    "@nx/angular": "16.0.0-beta.7"
```

Comment:
unsure what the deps policy is here, but the Nx migration pinned the deps. might need this switched out to `^<version>`